lxc_ssh.py

```diff
@@ -523,6 +523,19 @@ def __init__(self, play_context, new_stdin, *args, **kwargs):
         self.control_path = None
         self.control_path_dir = None
 
+    def _set_command_prefix(self):
+        # check for cgroupv2 and use systemd-run to run the commands if needed
+        (returncode_cgroup, stdout_cgroup, stderr_cgroup) = self._exec_command(
+            "ls /sys/fs/cgroup/cgroup.controllers", None, False
```

Comment:
Shouldn't that be a stat()?

Reply:
Yes, of course. This was... a very tacky implementation at the start when I was trying to figure out why lxc wasn't working.
Will push a new one soon, thank you!